### PR TITLE
Take master SD type from env var

### PR DESCRIPTION
**Pull request by acernek**
So that we can easily switch the type.

### CODE REVIEW

File: ost_utils/pytest/fixtures/storage.py [hunk 1]
```diff
@@ -34,3 +36,18 @@ def sd_nfs_host_storage_name():
...
+
+
+@pytest.fixture(scope="session")
+def secondary_storage_domain_name(master_storage_domain_type):
+    return constants.SD_ISCSI_NAME if master_storage_domain_type == "nfs" else constants.SD_NFS_NAME
+
+
+@pytest.fixture(scope="session")
+def master_storage_domain_type():
+    return os.environ.get("MASTER_SD_TYPE", "nfs")
```
Comment (tinez) [inline ost_utils/pytest/fixtures/storage.py:53]: We keep the things that come from the env in `fixtures/env.py`, so let's move this fixture there. The names for the env vars are also prefixed with `OST_`, so let's do the same here.

**Review by tinez — CHANGES_REQUESTED** [review]
Let's also use the new constant in `test_storage_domains` from UI tests.

File: ost_utils/pytest/fixtures/storage.py (hunk 1 above)
Comment (acernek) [reply ost_utils/pytest/fixtures/storage.py]: Sorry, missed that part. Done

**Review by tinez — APPROVED** [review]

**Pull request merged**